.editorconfig

```diff
@@ -13,6 +13,6 @@ trim_trailing_whitespace = true
 [*.bat]
 end_of_line = crlf
 
-[*.yml]
+[*.{yml,xml,json,ctp,css}]
```

Comment:
I thought we could define these standards for HTML and CSS, based on the [standards proposed by the creator of Bootstrap](http://mdo.github.io/code-guide/#html-syntax). I find it more readable with 2 spaces too.

We did not define standards for HTML and CSS so we have different usages, for example in [cake.css](https://github.com/cakephp/app/blob/master/webroot/css/cake.css) : 2 spaces and in [base.css](https://github.com/cakephp/app/blob/master/webroot/css/base.css) : 4 spaces.

I changed it for xml and json because we already adopt it for yml, so why not for these same sort of types?